homework9/src/pages/Confirmation/Confirmation.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import DescriptionText from "../../components/DescriptionText/DescriptionText";
+
+import "./Confirmation.css";
+
+export default function Confirmation() {
```

Comment:
that confirmation folder should be inside Apply folder. Confirmatin it is only component but not separate page